server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -236,6 +240,18 @@ export class TenantManager {
      * Retrieves the secret for the given tenant
      */
     public async getTenantKeys(tenantId: string, includeDisabledTenant = false): Promise<ITenantKeys> {
+        const lumberProperties = { [BaseTelemetryProperties.tenantId]: tenantId };
+
+        // Read from cache first
+        const cachedKey = await this.getCacheKey(tenantId);
+
+        if (cachedKey) {
+            Lumberjack.info(`Tenant key(s) found in cache`, lumberProperties);
```

Comment:
I like this idea. Instead of having the 2 log lines in line 243 and line 254, we would use a single Lumber metric, something like `Riddler.getTenantKeys`, and then add a property like `retrievedFrom` as Xin suggested. The property would have either one of these values: `cache`, `database`. And then, having the metric instead of logs would also help us understand performance/latency in these scenarios here, and how many cache hits vs misses we get.

Reply:
Added a RiddlerFetchTenantKey metric with the retrievedFrom property, which is specified based on cache or database retrieval